README.rst

```diff
@@ -138,14 +138,60 @@ Render your documentation
       Documentation-GENERATED-temp/Result/project/0.0.0/_buildinfo/warnings.txt
 
 
+4. Optional: Open the results in the default browser::
+
+      xdg-open "Documentation-GENERATED-temp/Result/project/0.0.0/Index.html"
```

Comment:
Up to here 4 spaces are used to indent code, below only 3 spaces are used.

Reply:
So what should be used in this file: 3 or 4?